tribuo-classification/enclave/src/main/java/com/r3/conclave/sample/enclave/MlEnclave.java

```diff
@@ -102,7 +99,7 @@ protected void collectData(InputData inputData) {
      */
     public static Object deserialize(byte[] inputData) {
         ByteArrayInputStream in = new ByteArrayInputStream(inputData);
-        ObjectInputStream is = null;
+        ObjectInputStream is;
```

Comment:
This method is printing the stack trace. I would suggest instead that you wrap the exception in a `RuntimeException` and throw that instead. That way this method no longer returns null, and the code in `receiveMail` no longer needs to do a null check. 

By throwing an exception here you also gain another benefit which is that `EnclaveClient.sendMail` will throw an `EnclaveException` with the same exception message. This makes it easier to debug during development.

Reply:
yes agreed. done.